tests/pcmsolver/ccsd-pted-heh+/input.dat

```diff
@@ -0,0 +1,64 @@
+#! coupled-cluster with PCM
+#! Perturbation-to-the-Energy-and-Density (PTED) algorithm
+
+
+nucenergy = 1.4084507042253522 # TEST
+scf_polenergy = -0.1538411942638307 # TEST
+scf_totalenergy = -3.0767011601643377 # TEST
+pted_polenergy = -0.000005991584194 # TEST
+pted_correnergy = -0.036646172163364 # TEST
+pted_totalenergy = -3.113347332327696 # TEST
+#! Reference values from DALTON PCM-FCI calculation
+fci_correnergy = -0.03664617216399968 # TEST
+fci_totalenergy = -3.113347332292 # TEST
+
+molecule HeH
+{
+	symmetry c1
+	H  0.00000 0.00000 0.00000
+	He 0.00000 0.00000 1.42000
+	1 1
+	units bohr
+	no_reorient
+	no_com
+}
+
+set
+{
+	basis cc-pVDZ
+	scf_type pk
+	pcm true
+	pcm_scf_type total
+	pcm_cc_type pted
+	d_convergence 12
+
+	e_convergence 12
+	r_convergence 12
+	ints_tolerance 20
+}
+
+pcm =
+{
+	Units = Angstrom
+	Medium
+	{
+		SolverType = IEFPCM
+		Solvent = Water
+	}
+	Cavity
+	{
+		Type = GePol
+		Area = 0.3
+		Mode = Implicit
+	}
+}
```

Comment:
I didn't know we allowed this, does this turn in a dictionary internally?

Reply:
It's a way to get to a text input file [this](https://github.com/psi4/psi4/blob/master/tests/pytest/test_addons.py#L406-L420) and [this](https://github.com/psi4/psi4/blob/eec3fd68d69b912a82274c916c3b0b30438302ba/psi4/driver/p4util/python_helpers.py#L230-L245) which, for consistency with other QC programs, is what PCMSolver actually takes.